src/main/java/com/gizasystems/automationexercise/pages/BrandProductsPage.java

```diff
@@ -0,0 +1,69 @@
+package com.gizasystems.automationexercise.pages;
+
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+
+public class BrandProductsPage {
+    private final SHAFT.GUI.WebDriver driver;
+    private final By productsbtn_button = By.xpath("//a[@href='/products']");
+    private final By brandsBody = By.xpath("//div[@class='brands_products']");
+    private final By brandNameBtn_button = By.xpath("//a[@href='/brand_products/Polo']");
+    private final By brandName = By.xpath("//div/h2[@class='title text-center']['Brand - Polo Products']");
+    private final By productsOfBrand = By.xpath("//a[@href='/product_details/1']");
+    private final By secondBrandName_button = By.xpath("//a[@href='/brand_products/H&M']");
+    private final By secondBrandName = By.xpath("//div/h2[@class='title text-center']['Brand - H&M Products']");
```

Comment:
Brand name should be test data not part of the xpath to be more dynamic.

Reply:
Solved.